ai_oca_bridge/models/ir_model.py

```diff
@@ -12,10 +12,17 @@ class IrModel(models.Model):
     ai_usage = fields.Char(store=False, search="_search_ai_usage")
 
     def _reflect_model_params(self, model):
-        vals = super(IrModel, self)._reflect_model_params(model)
-        vals["is_ai_bridge_thread"] = (
-            isinstance(model, self.pool["ai.bridge.thread"]) and not model._abstract
-        )
+        vals = super()._reflect_model_params(model)
+        if "ai.bridge.thread" in self.pool:
+            inherits = (
+                model._inherit
+                if isinstance(model._inherit, list)
+                else [model._inherit]
+                if model._inherit
+                else []
+            )
+            if "ai.bridge.thread" in inherits and not model._abstract:
```

Comment:
I don't understand why you keep this mechanism. You should remove it

Reply:
You can add as new commits the changes needed in the other tests AFAIK.

Reply:
:+1:  it is not usual, but sometimes we have to do it this way. There are other examples:

https://github.com/OCA/edi-framework/pull/192